packages/dashboard/src/customization/propertiesSections/gaugeStyleSettings/axisSettingsSection.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { Box, FormField, Input } from '@cloudscape-design/components';
+
+import { PropertyLens } from '~/customization/propertiesSection';
+import { GaugeProperties } from '~/customization/widgets/types';
+import { DashboardWidget } from '~/types';
+import {
+  FormLabel,
+  StyledExpandableSection,
+} from '../components/styledComponents';
+import { maybeWithDefault } from '~/util/maybe';
+
+export const AxisSettingsSection = ({
```

Comment:
instead of a new YAxis settings section could you reuse the one found at `dashboard/src/customization/propertiesSections/lineAndScatterStyleSettings/yAxis.tsx` 

you will need to move it out of its current folder under LineAndScatterStyles into its own and add in some logic to only show the Y-axis label input for the line chart